tests/test_workflow/test_methode_attribute.py

```diff
@@ -0,0 +1,124 @@
+import unittest
+from dessia_common.workflow.blocks import GetModelAttribute, SetModelAttribute
+from dessia_common.workflow.core import Variable, TypedVariable
+from dessia_common.typings import AttributeType
+from dessia_common.forms import DessiaObject
+from typing import List, Dict
+from parameterized import parameterized
+
+class DummyClassForTesting(DessiaObject):
+    def __init__(self, int_: int , variable_, float_: float = 0.0, str_: str = 'test', bool_: bool = True,
+                  l: List[int] = [1,2,3], d: Dict[str,int] = {'a':1,'b':2}, two: int = 2, name: str ='', **kwargs):
+        self.int_ = int_
+        variable_= variable_
+        self.float_ = float_
+        self.str_ = str_
+        self.bool_ = bool_
+        self.list_ = l
+        self.d = d
+        self.two = two
+        self.name = name
+        self.combine = self.int_*5
+        DessiaObject.__init__(self, **kwargs)
+    
+    def add_some_value(self) -> None:
+        self.combine = self.int_+self.two
+        return 
+
+
+class TestGetModelAttribute(unittest.TestCase):          
+    def setUp(self):
+        pass
+
+    @parameterized.expand([
+        (int, 'int_'),
+        (float, 'float_'),
+        (str, 'str_'),
+        (bool, 'bool_'),
+        (List[int], 'l'),
+        (Dict[str,int], 'd'),
+        (int, 'two'),
+        (str, 'name'),
+        (None, 'variable_'),
+        (None, 'combine'),
+        (None, 'kwargs')])
+    def test_outputs(self, type_, name):
+        GMA = GetModelAttribute(AttributeType(DummyClassForTesting,name))
```

Comment:
variables names should always be "lower_case_with_underscores" (also called snake_case) with at least 3 characters (see [python conventions](https://peps.python.org/pep-0008/#naming-conventions)).

Consider naming these variables (GMA, TV & V) as such. For ex `block`, `typed_variable` and `variable`